apps/fees-pay/ccpay-payment-api/perftest.yaml

```diff
@@ -6,6 +6,17 @@ spec:
   releaseName: ccpay-payment-api
   values:
     java:
+      replicas: 2
+      autoscaling:
+        enabled: true
+        maxReplicas: 6
+        memory:
+          enabled: true
+          averageUtilization: 90
+      memoryRequests: '1536Mi'
```

Comment:
In my opinion, this could be increased to 2Gb, otherwise the usage would not allow to scale down.

Reply:
Hi @rapolaskaseliscgi I've discussed this with Sammi, I think it's ok to have this around 1.5GB, that's still 500GB bigger than is was and memory usage never really increased above 50%.

Reply:
In that case scaling on memory could be disabled (or changed the avg. utilization parameter). With the numbers as they are, k8s would not scale down if it bumped more replicas. It probably would not scale up also, but it might happen, a peak or smth, and once it is up, it would not go down.